### PR TITLE
feat/uin-lp-off-boarding

### PR DESCRIPTION
**Pull request by nduchak**
# Description

# Contribution Checklist

- [ ] PR title starts with `(PE-<TICKET_NUMBER>)`
- [ ] Code approved
- [ ] Tests approved
- [ ] CI Tests pass

# Checklist

- [ ] Every contract variable/method declared as public/external private/internal
- [ ] Consider if this PR needs the `officeHours` modifier override
- [ ] Verify expiration (`30 days` unless otherwise specified)
- [ ] Verify hash in the description matches [here](https://emn178.github.io/online-tools/keccak_256.html)
- [ ] Validate all addresses used are in changelog or known
- [ ] Notify any external teams affected by the spell so they have the opportunity to review
- [ ] Deploy spell `ETH_GAS="XXX" ETH_GAS_PRICE="YYY" make deploy`
- [ ] Verify `mainnet` contract on etherscan
- [ ] Change test to use mainnet spell address and deploy timestamp
- [ ] Run `make archive-spell` or `make date="YYYY-MM-DD" archive-spell` to make an archive directory and copy `DssSpell.sol`, `DssSpell.t.sol`, `DssSpell.t.base.sol`, and `DssSpellCollateralOnboarding.sol`
- [ ] `squash and merge` this PR

### CODE REVIEW

File: src/test/config.sol [hunk 1]
```diff
@@ -100,9 +101,9 @@ contract Config {
         //
         spellValues = SpellValues({
             deployed_spell:                 address(0),        // populate with deployed spell if deployed
-            deployed_spell_created:         0,                 // use get-created-timestamp.sh if deployed
+            deployed_spell_created:         1654647217,                 // use get-created-timestamp.sh if deployed
```
Comment (hbarcelos) [inline src/test/config.sol:104]: I think this should be set to `0` if the spell was not deployed yet.

File: src/test/config.sol [hunk 2]
```diff
@@ -122,10 +123,10 @@ contract Config {
             flap_beg:              400,                     // in basis points
             flap_ttl:              30 minutes,              // in seconds
             flap_tau:              72 hours,                // in seconds
-            flap_lid:              150 * THOUSAND,          // in whole Dai units
+            flap_lid:              30 * THOUSAND,           // in whole Dai units
             cat_box:               20 * MILLION,            // In whole Dai units
             dog_Hole:              100 * MILLION,           // In whole Dai units
-            esm_min:               100 * THOUSAND,          // In whole MKR units
+            esm_min:               150 * THOUSAND,          // In whole MKR units
```
Comment (hbarcelos) [inline src/test/config.sol:129]: Why are we changing those?

File: src/test/config.sol (hunk 1 above)
Comment (nduchak) [reply src/test/config.sol:104]: Not matter at all

**Comment by nduchak** [comment]
Let me move this spell to goerli spells first then I will reopen this one.

**Pull request closed without merging**